### PR TITLE
Update DTable Doxygen Comments

### PR DESCRIPTION
**Pull request by kathlenehurt-sifive**
Added a description of how the DTable traversal works.

### CODE REVIEW

File: mavis/DTable.h [hunk 1]
```diff
@@ -39,6 +39,58 @@ namespace mavis {
...
+ * For this example, the form "AMO" defines the following opcode fields:
+ *   - opcode (bits 0:6)
+ *   - func3 (bits 12:14)
+ *   - func5 (bits 27:31)
+ *
+ * | 31|   |   |   | 27| 26| 25| 24|   |   |   | 20| 19|   |   |   | 15| 14|   | 12| 11|   |   |   |  7|  6|   |   |   |   |   |  0|
+ * |:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|
+ * |**func5**||||| | |**rs2**|||||**rs1**|||||**func3**|||**rd**|||||**opcode**|||||||
+ *
+ * The Stencil provides the fixed values of the opcode fields. For an isntruction to be deocded as
```
Comment (klingaard) [inline mavis/DTable.h:68]: spelling typo `isntruction`

**Review by klingaard — APPROVED** [review]
That's awesome documentation!

Will you be documenting the other fields that are available to use?  For example, there are `ignore` fields, etc..

File: mavis/DTable.h (hunk 1 above)
Comment (kathlenemagnus) [reply mavis/DTable.h:68]:
```suggestion
 * The Stencil provides the fixed values of the opcode fields. For an instruction to be decoded as
```

**Comment by kathlenemagnus** [comment]
> That's awesome documentation!
> 
> Will you be documenting the other fields that are available to use? For example, there are `ignore` fields, etc..

Eventually we should, yes. But I'm not planning to any time soon.

**Pull request merged**